### PR TITLE
adds the test cases for runs/console page

### PR DESCRIPTION
**Pull request by ankunath**
https://github.com/Shippable/bat/issues/349

### CODE REVIEW

File: tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+var Shippable = require('../../../../_common/shippable/Adapter.js');
+
+var testSuite = util.format('%s2 - %s', testSuiteNum,
+  testSuiteDesc);
+
+var isTestFailed = false;
+var testCaseErrors = [];
+var runId;
+var run;
+var jobs = {};
```
Comment (jatindogra) [inline tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js]: it should be declared as an array.

File: tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+            this.timeout(0);
+            var query = util.format('runIds=%s',runId);
+            shippable.getRuns(query,
+              function (err, res) {
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
+                       '\n - [ ] %s get runById failed for runId: $s'+
+                      'with error: %s' + testSuiteDesc, runId, err);
```
Comment (nranjanShippable) [inline tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js]: Use `,` for `util`.

File: tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+            this.timeout(0);
+            runId = nconf.get('shiptest-GITHUB_ORG_1:runId');
+            var query = 'runIds=' + runId;
+            shippable.getJobs(query,
+              function (err, resJobs) {
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
+                      '\n - [ ] %s get jobs failed with error: %s for runId: %s' +
```
Comment (nranjanShippable) [inline tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js]: Use `,` for `util`.

File: tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+              type: 'push'
+            };
+            shippable.triggerNewBuildByProjectId(projectId, payload,
+              function (err, msg) {
+                if (err) {
+                  isTestFailed = true;
+                  var testCase =
+                    util.format(
+                      '\n - [ ] %s trigger new build failed for projectId: $s'+
+                      'with error: %s' + testSuiteDesc, projectId, err);
```
Comment (nranjanShippable) [inline tests/organization-owner/github/5-projects/1.2.10.RunsConsole.js]: Use `,` instead of `+`

File: tests/getSystemCodes.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            }
+          }
+        );
+      }
+    );
+
+    it('Should save systemConfigs to config file',
+      function (done) {
+        nconf.set('shiptest-github-owner:systemCodes',systemCodes);
+        nconf.save(function(err){
```
Comment (nranjanShippable) [inline tests/getSystemCodes.js]: space before `{`

File: tests/getSystemCodes.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+              return done();
+            }
+          }
+        );
+      }
+    );
+
+    it('Should save systemConfigs to config file',
+      function (done) {
+        nconf.set('shiptest-github-owner:systemCodes',systemCodes);
```
Comment (nranjanShippable) [inline tests/getSystemCodes.js]: space after `,`

**Pull request merged**